zbus_macros/src/iface.rs

```diff
@@ -1089,6 +1115,36 @@ fn introspect_method(name: &str, args: &TokenStream) -> TokenStream {
     )
 }
 
+fn is_special_arg(attrs: &[Attribute]) -> bool {
```

Comment:
nit: I've been trying to follow top-down code layout approach, which means use before definition. If you could just move it down after its first use, that would be great.